packages/odyssey-react-mui/src/Form.tsx

```diff
@@ -17,6 +17,14 @@ import { Button } from "./Button";
 import { Infobox } from "./Infobox";
 import { useUniqueId } from "./useUniqueId";
 
+export const formEncodingTypeValues = [
+  "application/x-www-form-urlencoded",
+  "multipart/form-data",
+  "text/plain",
+] as const;
```

Comment:
I would expect `'application/json'` to be in this list. Not sure where we came up with this list. Did we get it from somewhere?

Reply:
These are the only three supported by HTML's `enctype`.